react-typescript.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    jest: true,
+  },
+  extends: 'airbnb',
+  plugins: ['react'],
+  rules: {
+    curly: ['error', 'multi'],
+    indent: ['error', 2],
+    'jsx-a11y/label-has-associated-control': ['error', { assert: 'either', depth: 5 }],
+    'no-param-reassign': 'off',
+    'object-curly-newline': 'off',
+    'react/react-in-jsx-scope': 'off',
+    'react/require-default-props': 'off',
+    'max-len': ['error', 150, 2, {
```

Comment:
Why 150 characters? I am strongly against having lines longer than 80 characters (except in some situations, but those are exceptions that can be lint ignored and not rules).

Reading longer lines makes very difficult to review code (especially in mobile devices), makes very annoying to have two different code files in a same display side-by-side, and it is a readibility hell to have long lines.